Add basic tests

ahalterman opened the pull request:
Add two basic tests of the pipeline with the `pytest` framework. Future modifications to the pipeline should come with associated tests of their functionality and shouldn't break existing tests.

These tests can be run by running `py.test` from the command line inside the `tests/` directory. `pip install pytest` if it's not installed.

johnb30 commented:
Couple things:

1) Might want to add some line breaks/formatting into those long lines. Just for readability.

2) While you're writing tests you might consider setting up Travis CI integration. I might need to be the one to turn that on, but if you setup the `.travis.yml` file I'll turn it on.

ahalterman commented:
Added the `.travis.yml` file and cleaned up the formatting by putting the different fields on their own lines. I didn't want to mess with formatting each string, so it's not perfectly 80 characters.

johnb30 commented:
👍
